templates/repo/issue/labels/edit_delete_label.tmpl

```diff
@@ -29,21 +29,24 @@
 			<div class="required field">
 				<label for="name">{{.locale.Tr "repo.issues.label_title"}}</label>
 				<div class="ui small input">
-					<input class="new-label-input emoji-input" name="title" placeholder="{{.locale.Tr "repo.issues.new_label_placeholder"}}" autofocus required maxlength="50">
+					<input class="label-name-input emoji-input" name="title" placeholder="{{.locale.Tr "repo.issues.new_label_placeholder"}}" autofocus required maxlength="50">
 				</div>
 			</div>
-			<div class="field">
+			<div class="field label-exclusive-input-field">
 				<div class="ui checkbox">
-					<input class="new-label-exclusive" name="exclusive" type="checkbox">
+					<input class="label-exclusive-input" name="exclusive" type="checkbox">
 					<label>{{.locale.Tr "repo.issues.label_exclusive"}}</label>
 				</div>
 				<br/>
 				<small class="desc">{{.locale.Tr "repo.issues.label_exclusive_desc" | Safe}}</small>
+				<div class="desc gt-hidden label-exclusive-warning">
+					<br/>{{.locale.Tr "repo.issues.label_exclusive_warning" | Safe}}
```

Comment:
This looks fairly hacky. What about
```suggestion
				<div class="desc gt-ml-2 gt-mt-3 gt-hidden label-exclusive-warning">
					{{svg "octicon-alert"}} {{.locale.Tr "repo.issues.label_exclusive_warning" | Safe}}
```
instead?
This produces the following result:
![image](https://user-images.githubusercontent.com/51889757/219877385-2715c86e-a8da-4785-b7c2-cdbf92f50bb6.png)